chart-parts/ingress.yaml

```diff
@@ -31,14 +32,19 @@ data:
 apiVersion: "extensions/v1beta1"
 kind: "Ingress"
 metadata:
-  name: "ingress-scf"
-  namespace: {{ .Release.Namespace | quote }}
+  name: "{{ .Release.Name }}-{{ .Values.services.ingress.class }}"
+  namespace: "{{ .Release.Namespace }}"
```

Comment:
Same question as for https://github.com/SUSE/uaa-fissile-release/pull/142/files, prefer to stay with `| quote` as the dominant way for us quoting a single value.

Reply:
Answered on the referenced PR.